src/components/Resume/ResumeDetail/ResumeDetail.jsx

```diff
@@ -0,0 +1,124 @@
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import React, { useState } from "react";
+import { GetUser } from "./ResumeApi";
+import "./ResumeDetail.css";
+import { useEffect } from "react";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: 500,
+  },
+  margin: {
+    "& > *": {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+
+function Resume({ userId }) {
+  const [user, setUser] = useState();
+
+  const refresh = () => {
+    let userRes;
+    GetUser(userId)
+      .then((result) => {
+        userRes = result;
+      })
+      .then(() => setUser(userRes));
+  };
+  useEffect(refresh, [userId]);
+
+  const [readOnly, setReadOnly] = useState(true);
+  const [showEdit, setShowEdit] = useState("Edit");
+  const classes = useStyles();
+
+  const edit = (event) => {
+    event.preventDefault();
+    setReadOnly(!readOnly);
+    if (readOnly) {
+      setShowEdit("Save");
+      alert("You can edit");
+    } else {
+      setShowEdit("Edit");
+      alert("Saved");
```

Comment:
these alerts are good for now. They don't provide a good user experience so **we are removing them as soon as possible**.
Please leave some comments in your code about why we are using those alerts right now and that they are going to be replaced.